src/StochasticDiffEq.jl

```diff
@@ -47,6 +47,8 @@ module StochasticDiffEq
 
   import DiffEqBase: iip_get_uf, oop_get_uf, build_jac_config
 
+  import SparseDiffTools: forwarddiff_color_jacobian!, ForwardColorJacCache
```

Comment:
I guess `using` is more appropriate? (see https://docs.julialang.org/en/v1/manual/modules/#Summary-of-module-usage-1)